src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -140,9 +138,18 @@ def generate_initial_population(self, pop_data: SimulantData) -> None:
             np.round(pop["age"] * 365.25), unit="days"
         )
 
-        # Add Social Security Numbers
-        pop["ssn"] = self.ssn_generator.generate(pop).ssn
-        pop["ssn"] = self.ssn_generator.remove_ssn(pop["ssn"], self.proportion_with_no_ssn)
+        # Deterimne if simulants have SSN
+        pop["ssn"] = False
+        # Give native born simulants a SSN
+        pop.loc[pop["born_in_us"], "ssn"] = True  # Give simulants born in US a SSN
+        # Choose which non-native simulants get a SSN
+        non_natives_idx = pop.loc[
+            ~pop["born_in_us"]
+        ].index  # Get index for sims born outside of United States
```

Comment:
If you have your comments on separate lines they won't screw up the code formatting as much when you run black.

Also, the syntax below is more succinct, does the same thing, and enables you just to get the (non-)native-born index once.
```suggestion
        native_born_idx = pop.index[pop["born_in_us"]]
        # Give simulants born in US a SSN
        pop.loc[native_born_idx, "ssn"] = True
        
        # Choose which non-native simulants get a SSN
```

You'd need to rename `non_natives_idx` below.